nx-X11/programs/Xserver/hw/nxagent/Screen.c

```diff
@@ -3693,270 +3705,54 @@ Bool intersect_bb(int ax1, int ay1, unsigned int aw, unsigned int ah,
      */
 
     /* check if outside-left of bounding box */
-    if (ax1 < bbx1) {
+    if (bx1 == bbx1 && ax1 - bbx1 < 0) {
```

Comment:
are you sure that "-" takes precendence over "<"?  I'd expect this code too look like this:

if (bx1 == bbx1 && (ax1 - bbx1) < 0) {

Reply:
Ok, looks like this is correct (see http://en.cppreference.com/w/c/language/operator_precedence). Still I'd like it to be enclosed...